gui/src/components/loaders/BlinkingDot.tsx

```diff
@@ -0,0 +1,34 @@
+import styled, { css, keyframes } from "styled-components";
+
+const DEFAULT_DIAMETER = 8;
+
+const blink = keyframes`
+  0%, 100% {
+    transform: scale(1);
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.25;
+  }
+`;
+
+const blinkAnimation = css`
+  animation: ${blink} 1s infinite;
```

Comment:
1s animation felt pretty fast to me. I tried 3s and it felt less intense - let me know your thoughts though.

```suggestion
  animation: ${blink} 3s infinite;
```